crates/swc_ecma_parser/src/lib.rs

```diff
@@ -229,7 +235,11 @@ impl Syntax {
     /// Should we parse typescript?
     #[cfg(feature = "typescript")]
     pub const fn typescript(self) -> bool {
-        matches!(self, Syntax::Typescript(..))
+        match self {
+            #[cfg(feature = "typescript")]
```

Comment:
Please remove this line

Reply:
apologies, overlooked the already present attribute